lib/src/components/data_table/data_table.dart

```diff
@@ -74,10 +79,16 @@ class SkawaDataTableComponent implements OnDestroy {
   @Output('highlight')
   Stream<RowData> get onHighlight => _highlightController.stream;
 
+  @Output()
```

Comment:
the output should be `'sort'`

Reply:
Funny, this is what I did originally, but I encountered with a strange naming collision for `sort`, with the output, the directive input and the data table SortModel variable name all being identical. But you're right, this should be `sort`, so I renamed the directive input to `sortable` and the `SortModel` variable to `sortModel`. Fixed.